app/views/play/level/modal/CourseVictoryComponent.vue

```diff
@@ -28,7 +28,7 @@
               div {{ $t('play_level.combo_challenge_complete_body', { concept: primaryConcept }) }}
                   
             div.clearfix.well.well-sm.well-parchment(v-else-if="assessmentNext")
-              img.lock-banner(src="/images/pages/play/modal/lock_banner.png")
+              img.lock-banner(src="/images/pages/play/modal/unlocked_banner.png")
```

Comment:
Why this change?

Reply:
I renamed this image to be `unlocked_banner` since it is for unlocked state, and there is another image for locked status which is `locked_banner`. So I renamed it to make it clear.